### PR TITLE
Intents to open a file and download a file

### PR DESCRIPTION
**Pull request by ptbrowne**
Downloading a file is useful for example for the EDF app when you want to download your bills.
It also can be useful to view your files in an iframe.

### CODE REVIEW

File: src/services.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+  cozy.client.init({
+    cozyURL: '//' + data.cozyDomain,
+    token: data.cozyToken
+  })
+
+  render((
+    <I18n context={context} lang={data.cozyLocale}>
+      <Provider store={store}>
+        <IntentService window={window} />
```
Comment (enguerran) [inline src/services.jsx]: I don't think that it is useful to pass a global object into another scope.

File: src/services.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  require('preact/devtools')
+
+  // Export React to window for the devtools
+  window.React = React
+}
+
+// Enable Redux dev tools
+const composeEnhancers = (__DEVELOPMENT__ && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose
+
+const store = createStore(
```
Comment (enguerran) [inline src/services.jsx]: There is no need of a redux store in this sub-application.

File: src/containers/IntentService.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+import styles from '../styles/services'
+
+import React, { Component } from 'react'
+
+import Loading from '../components/Loading'
+import ServiceBar from '../components/services/ServiceBar'
+import ViewerService from '../components/services/ViewerService'
+
+export default class IntentService extends Component {
```
Comment (enguerran) [inline src/containers/IntentService.jsx]: I guess we could write something simpler than this very big and not designed component.

**Comment by ptbrowne** [comment]
Better PR here : https://github.com/cozy/cozy-drive/pull/269

**Pull request closed without merging**